cluster/validate-cluster.sh

```diff
@@ -112,7 +112,7 @@ while true; do
   node="${node%.}"
   if [ "${res}" -ne "0" ]; then
     if [[ "${attempt}" -gt "${last_run:-$MAX_ATTEMPTS}" ]]; then
-      echo -e "${color_red} Failed to get nodes.${color_norm}"
+      echo -e "${color_red:?} Failed to get nodes.${color_norm:?}"
```

Comment:
imo these colors are not important, I'd about say we should just do `${color_red:-}` instead because the script will still work without them

Reply:
I agree. I will change it.
Other files have the same definition. Should we change in other files, too?

Reply:
yeah, I can't think of a case where we'd actually want to crash on these

Reply:
I changed it. I will modify other files with other PRs.